communicator_node:
  ros_parameters:
    robot_classes:
      - Husky
      - Kobuki
      - Locobot
